client/lib/react-test-env-setup/index.js

```diff
@@ -45,3 +45,13 @@ module.exports.cleanup = function() {
 	delete global.localStorage;
 	delete global.XMLHttpRequest;
 };
+
+module.exports.auto = function( markup, features ) {
```

Comment:
Ideally we should expose only this function, then we could leave it anonymous. We could also we move it to `client/test/react-env-setup.js` and include directly from there.